src/useRefs/useRefs.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta } from '@storybook/addon-docs';
```

Comment:
We should also include documentation for `arrayRef`

Reply:
Added docs for `arrayRef`